src/components/Holdings.tsx

```diff
@@ -1,15 +1,15 @@
 import { css } from "@emotion/react"
+import Head from "next/head"
```

Comment:
My vscode is configured to automatically sort and prune the imports which is why they changed order and some unused imports were removed in a few files

Reply:
it requires a plugin. This is a common one I've used in a few places: https://github.com/trivago/prettier-plugin-sort-imports